### DISCUSSION TITLE
Conversion: tab shows 1/1 and then 2/11

### DISCUSSION THREAD

**Discussion by HubKing** [google/mozc#705] [Q&A] [answered]
I normally use only the space bar for conversion. This time, I tried to write `ご無用`, but when I typed `ごむよう` and pressed the space bar, it only showed `ゴム用`. I tried to find a way to convert it to `ご無用` with arrow keys and space bar, but failed to find a way. Then, I randomly thought of pressing tab, like common IDE's. 

The strange thing is, at first, it only showed `ご無用 [1/1]` and then when I pressed tab again, the index became `2/11` and the second candidate was `ご無用`. Why did it show `1/1` at the first tab, not `1/11`?

Also, was tab the best way to get `ご無用` in this case? I mean, could I get it with the space bar?

[Screencast from 2023-04-02 01-07-13.webm](https://user-images.githubusercontent.com/10900489/229302344-2c84f3ea-fade-4307-bdcc-462685445c1c.webm)

**Comment by hiroyuki-komatsu** [comment, marked as answer]
Hi HubKing,

Those numbers come from the current behavior. When you press `tab`, you will see candidates up to 3 (e.g. [1/1] in your case) with fast logic. When you press `tab` again, you will see all candidates with full logic.

For the interaction after spacebar, `Shift Right` and `Shift Left` are shortcuts to resize the focus area.

**Reply by HubKing** [reply]
Yeah, shift+arrow worked **after** I converted it to ゴム用. I had tried to designate the range **before** the initial conversion.  So, I guess I must do the initial wrong conversion to designate the range.

https://user-images.githubusercontent.com/10900489/229358268-7106349e-04b9-436b-b6ff-17ae827d14cf.mp4
